server2/Arcadia.Assistant/Arcadia.Assistant.PushNotifications/WellKnownBuildTypes.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Arcadia.Assistant.PushNotifications
+{
+    public static class WellKnownBuildTypes
```

Comment:
a) if you need device type in notifications - call it device notification fucking type, not build type, which is just a configuration name specific for appcenter
b) do not touch android/ios notification yet please, I've asked it number of times already

Reply:
for demonstration its enough to send two notifications on two different list of employees - ones with android and ones with ios devices.

Reply:
Current solution when you need to know parameter names is ugly. You just proved it with strings duplication. Keep it simple for now, we'll deal with it later

Reply:
Changed the name